generate_contexts_config.sh

```diff
@@ -56,6 +58,8 @@ replace_values "{AWS_BUCKET_USER_FOLDER}" "$AWS_BUCKET_USER_FOLDER"
 replace_values "{CA}" "$ca_pem_base64"
 replace_values "{CLIENT_CERT}" "$client_cert_base64"
 replace_values "{CLIENT_KEY}" "$client_key"
+replace_values "{CLOUD_USER_ID}" "$cloud_user_id"
+replace_values "{CLOUD_INSTANCE_ID}" "$UUID"
```

Comment:
@laurentluce i am not sure how this repo gets tested. this is best effort bash writing; i have verified that `cloud_user_id="${AWS_BUCKET_USER_FOLDER##*-}"` this replacement works